### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by russey**
GitHub actions and port LaTeX vignettes to RMarkdown

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+YEAR: 2015
+COPYRIGHT HOLDER: Mango Solutions
```
Comment (KKulma) [inline LICENSE:2]: is this an MIT license?

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,7 @@
+New in version 0.3.3:
+- Rework broken LaTeX vignettes into RMarkdown, removing LaTeX dependency.
+- Add R CMD check to GitHub actions
```
Comment (KKulma) [inline NEWS:3]: I'm not sure if we need to mention CI and GH Actions here as they don't constitute new funtionality in the packge

**Comment by KKulma** [comment]
great job, @russey, massive thanks for your help! It looks like the vignettes are still failing to build on Linux machines, I'll have a look if something stands out

File: vignettes/BLCOP.Rmd [hunk 3]
```diff
@@ -210,10 +210,10 @@ univariate distributions in a similar way and is used to create the view distrib
...
 
 ```{r, echo=TRUE, print=FALSE}
-pick <- matrix(0, ncol = 4, nrow = 1,
-	dimnames = list(NULL, c("SP", "FTSE", "CAC", "DAX")))
-pick[1,"DAX"] <- 1
-viewDist <- list(distribution("unif", min = -0.02, max = 0))
+# pick <- matrix(0, ncol = 4, nrow = 1,
+# 	dimnames = list(NULL, c("SP", "FTSE", "CAC", "DAX")))
+# pick[1,"DAX"] <- 1
+# viewDist <- list(distribution("unif", min = -0.02, max = 0))
```
Comment (KKulma) [inline vignettes/BLCOP.Rmd]: @russey If we're not producing an output, is it worth just adding `eval = FALSE` to the code chunk? I suspect CRAN may not like commetned out code in the vignette

**Comment by KKulma** [comment]
it all looks great, thanks, @russey !!! @Avatoo I'd be happy to merge it , but you probably want to have a look it first 👍

**Comment by Avatoo** [comment]
There is one trivial note when running `R CMD check --as-cran BLCOP_0.3.3.tar.gz`. Otherwise, great job both :)

Happy to merge it.

**Pull request merged**